### DISCUSSION TITLE
Indicate apps that are open but don't have any windows

### DISCUSSION THREAD

**Discussion by lawand-dot-io** [lawand-dot-io/taskbar#70] [Feature requests]
Find a way to indicate apps that are open but don't have any open windows. In the dock, this is done by adding a black dot which indicates that this app is currently open. This feature could be only for apps that are added to Taskbar using the "Keep in Taskbar" option, or it could be for all apps (so, similar to the Dock)

Update: When you close the last window of an app it's not possible to quit that app using Taskbar and the default Dock must be used instead. If apps that don't have open windows are indicated, an option can be added to quit those apps.

**Comment by lawand-dot-io** [comment]
This request was split from: https://github.com/lawand-dot-io/taskbar/issues/68

**Comment by adityakurian** [comment]
I personally like that it does not do this.

**Reply by lawand-dot-io** [reply]
yeah, if I ended up implementing this I will give users the choice whether or not to enable it 👍

**Comment by lawand-dot-io** [comment]
Comments from https://github.com/yo1995 in https://github.com/lawand-dot-io/taskbar/discussions/84

Currently if an app has all its windows closed, the taskbar will hide the app. It would be good to consider

- adding an option such as "Quit app when last window closes"
- collapse the apps that don't have open windows into icons (maybe with a highlighted underline indicating that it is running in the background)

**Reply by lawand-dot-io** [reply]
@yo1995: "Quit app when last window closes" is an idea I am already implementing as another app. I should be ready soon :) I'll keep your other idea in mind.

**Comment by lawand-dot-io** [comment]
Update: I published a second app which quits apps when their last window is closed, this solves one of the issues mentioned in this ticket

https://lawand.io/last-window-quits/

**Comment by jrking78** [comment]
I would like this feature. And it would be nice if the dot shows under the icon, and then you close the app, it would not move over to the left like it does now. It would just int he same location and the dot would be the indicator that it is open or not.

**Reply by lawand-dot-io** [reply]
I understand what you mean. Will keep this in mind. Cheers
